services/auth/auth.go

```diff
@@ -32,6 +32,11 @@ func isAttachmentDownload(req *http.Request) bool {
 	return strings.HasPrefix(req.URL.Path, "/attachments/") && req.Method == "GET"
 }
 
+// isFeed checks if the request targets a rss/atom feed
+func isFeed(req *http.Request) bool {
+	return strings.HasSuffix(req.URL.Path, ".rss") || strings.HasSuffix(req.URL.Path, ".atom")
```

Comment:
This lacks sufficient restrictions.

Reply:
Not worse than old code like `isAttachmentDownload` and `isContainerPath`

Reply:
Will do some improvements later.